Modules/install-jaspBase.R.in

```diff
@@ -19,7 +19,7 @@ if (Sys.info()["sysname"] == "Darwin" || Sys.info()["sysname"] == "Windows") {
 computeHash <- function(modulePkg) {
   srcFiles <- c(
     list.files(modulePkg,                    recursive=TRUE, full.names = TRUE, pattern = '(NAMESPACE|DESCRIPTION)$'),
-    # list.files(file.path(modulePkg, 'src'),  recursive=TRUE, full.names = TRUE, pattern = '(\\.(cpp|c|hpp|h)|(Makevars|Makevars\\.win))$'),
+    list.files(file.path(modulePkg, 'src'),  recursive=TRUE, full.names = TRUE, pattern = '(\\.(cpp|c|hpp|h)|(Makevars|Makevars\\.win))$'),
```

Comment:
maybe we should add `Makevars.ucrt` as well? 
https://developer.r-project.org/Blog/public/2021/03/12/windows/utf-8-toolchain-and-cran-package-checks/

> R now understands .ucrt suffix of Makevars and Makefile files.

Reply:
Good idea!